db/create_tables.sql

```diff
@@ -55,6 +55,8 @@ CREATE TABLE IF NOT EXISTS `state_machines` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `name` VARCHAR(255) NOT NULL,
   `version` TINYINT UNSIGNED NOT NULL,
```

Comment:
TINYINT is capable of storing up to 255. That's a big number for version column IMO considering each version would be having multiple SMs with unique internal Ids.
If you think it may cross, then we can go for next class SMALL INT with number till 65535.